src/renderer/components/MessageWrapper.js

```diff
@@ -241,16 +242,19 @@ function convertMessageStatus (s) {
 }
 
 function convertContentType (message) {
-  var filemime = message.filemime
-  if (!filemime) return 'image/jpg'
```

Comment:
I don't think we should return `image/jpg`. It should fallback to `application/octet-stream`.

Reply:
yeah makes sense, this was a hacky first pass, your solution is nice!